Resources/Prototypes/_Impstation/Objectives/traitor.yml

```diff
@@ -0,0 +1,99 @@
+#SVS objectives
+
+- type: entity
+  parent: [BaseTraitorObjective, BaseKillObjective]
+  id: KillRandomTraitorSvSObjective # Used for rounds when basically every valid person will be a traitor.
+  description: Do it however you like, just make sure they don't make it to centcom.
+  components:
+  - type: Objective
+    difficulty: 1.75
+    unique: true
+  - type: TargetObjective
+    title: objective-condition-kill-person-title
+  - type: PickRandomTraitor
+  - type: ObjectiveBlacklistRequirement
+    blacklist:
+      component: null
+
+- type: entity
+  parent: [BaseTraitorSvSSocialObjective, BaseKeepAliveObjective]
+  id: RandomTraitorAliveSvSObjective # Used for rounds when basically every valid person will be a traitor.
+  description: Identify yourself at your own risk. We just need them alive.
+  components:
+  - type: Objective
+    difficulty: 1.75
+    unique: true
+  - type: TargetObjective
+    title: objective-condition-other-traitor-alive-title
+  - type: RandomTraitorAlive
+  - type: ObjectiveBlacklistRequirement
+    blacklist:
+      component: null
+
+#steal
+
+- type: entity
+  parent: BaseTraitorStealObjective
+  id: FinFinMeatStealObjective
+  components:
+  - type: ObjectiveLimit
+    limit: 4
+  - type: StealCondition
+    stealGroup: FoodMeatFinfin
+
+- type: entity
+  parent: BaseTraitorStealObjective
+  id: MedTekCartridgeTraitorStealObjective
+  components:
+  - type: NotJobRequirement
+    job: MedicalDoctor
+  - type: StealCondition
+    stealGroup: MedTekCartridge
+  - type: Objective
+    difficulty: 1
+
+- type: entity
+  parent: BaseTraitorStealObjective
+  id: LogProbeCartridgeTraitorStealObjective
+  components:
+  - type: NotJobRequirement
+    job: Detective #you know. just in case
+  - type: StealCondition
+    stealGroup: LogProbeCartridge
+  - type: Objective
+    difficulty: 1
+
+- type: entity
+  parent: BaseTraitorStealObjective
+  id: AstroNavCartridgeTraitorStealObjective
+  components:
+  - type: NotJobRequirement
+    job: SalvageSpecialist
+  - type: StealCondition
+    stealGroup: AstroNavCartridge
+  - type: Objective
+    difficulty: 1
+
+- type: entity
+  parent: BaseTraitorStealObjective
+  id: WantedListCartridgeTraitorStealObjective
+  components:
+  - type: NotJobRequirement
+    job: SecurityOfficer
+  - type: StealCondition
+    stealGroup: WantedListCartridge
+  - type: Objective
+    difficulty: 1
+
+- type: entity
+  parent: BaseTraitorStealObjective
+  id: ShiningSpringTraitorStealObjective
```

Comment:
same as the pda cartridges: heads sign up for the role with the understanding that they're a high-value target, janitors should not be subject to "the antag needed to kill me to grab my high-value item"